src/deps.ts

```diff
@@ -1,4 +1,4 @@
-export { deepMerge } from "https://deno.land/std@0.134.0/collections/mod.ts";
+export { deepMerge } from "https://deno.land/std@0.134.0/collections/deep_merge.ts";
 export { parse } from "https://deno.land/std@0.134.0/flags/mod.ts";
 export { ensureDir } from "https://deno.land/std@0.134.0/fs/mod.ts";
 export * as path from "https://deno.land/std@0.134.0/path/mod.ts";
```

Comment:
I understand that this is the only module throwing the warning, but why don't we also have to do this for the other modules we import? Is it because this one is doing `export *` all over?

Reply:
I think it's based on how the collections [top-level `mod.ts` in STD is built](https://github.com/denoland/deno_std/blob/6e26b89f1b16daa1ffb70276aa4ee5971fe38105/collections/mod.ts). `mod.ts` exports _all the things_, so if you import `mod.ts`, you are also importing all the things. I don't think tree-shaking applies here.